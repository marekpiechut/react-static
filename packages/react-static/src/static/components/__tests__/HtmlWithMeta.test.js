import React from 'react'
import renderer from 'react-test-renderer'
import makeHtmlWithMeta from '../HtmlWithMeta'

describe('HtmlWithMeta', () => {
  test('when route is a static route', async () => {
    const HtmlWithMeta = await makeHtmlWithMeta({
      head: { htmlProps: { lang: 'en' } },
      plugins: [],
    })

    const htmlWithMeta = renderer
      .create(
        <HtmlWithMeta className="body">
          <head>
            <title>React Static</title>
          </head>
        </HtmlWithMeta>
      )
      .toJSON()

    expect(htmlWithMeta).toMatchSnapshot()
  })
})
