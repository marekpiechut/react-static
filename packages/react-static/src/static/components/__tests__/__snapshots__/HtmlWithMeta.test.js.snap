// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HtmlWithMeta when route is a static route 1`] = `
<html
  className="body"
  lang="en"
>
  <head>
    <title>
      React Static
    </title>
  </head>
</html>
`;
